ecs-cli/integ/cmd/compose.go

```diff
@@ -85,6 +113,34 @@ func TestServicePs(t *testing.T, p *Project, wantedNumOfContainers int) {
 	t.Logf("Project %s has %d running containers", p.Name, wantedNumOfContainers)
 }
 
+// TestTaskScale runs `ecs-cli compose scale` for a project.
+func TestTaskScale(t *testing.T, p *Project, scale int) {
+	args := []string{
+		"compose",
+		"--file",
+		p.ComposeFileName,
+		"--ecs-params",
+		p.ECSParamsFileName,
+		"--project-name",
+		p.Name,
+		"scale",
+		strconv.Itoa(scale),
+		"--cluster-config",
+		p.ConfigName,
+	}
+	cmd := integ.GetCommand(args)
+
+	// When
+	out, err := cmd.Output()
+	require.NoErrorf(t, err, "Failed to scale task", "error %v, running %v, out: %s", err, args, string(out))
+
+	// Then
+	stdout.Stdout(out).TestHasAllSubstrings(t, []string{
+		"Started container",
```

Comment:
I wish there was an easy way to guarantee that the right number of tasks are running.

I think this is fine for now though.

Reply:
I see this is done using PS in the actual integ test 👍🏻